packages/react-ui/src/components/button/index.js

```diff
@@ -5,19 +5,24 @@ import { styles } from './button.styles'
 import { merge } from '../../utils'
 
 /** Description of a button */
-const Button = ({ size, css, ...props }) => {
+const Button = ({ fullWidth, css, ...props }) => {
+  
+  let width = 'auto'
```

Comment:
I think we should respect the user prop as well, if user is trying to pass a custom width over here that should be prioritise.
so, taking the width from Buttons prop always which is `auto` by default. This makes more sense to me.

Reply:
It will

The way to pass a custom width is `css={{width: '50px'}}` which will override both 'auto' and 'fullWidth'

https://github.com/siddharthkp/react-ui/pull/36/files#diff-f82e908416c0d5d7d7f255ad314a6cfaR17

Reply:
yeah right.